src/bun.js/bindings/bindings.zig

```diff
@@ -6780,6 +6790,14 @@ pub fn toJSHostFunction(comptime Function: JSHostZigFunction) JSC.JSHostFunction
 
 // XXX: temporary
 pub fn toJSHostValue(globalThis: *JSGlobalObject, value: error{ OutOfMemory, JSError }!JSValue) JSValue {
+    if (bun.Environment.allow_assert) {
```

Comment:
Let's avoid making function calls slower on Windows

```suggestion
    if (bun.Environment.isDebug) {
```